sdk/resourcemanager/azure-resourcemanager-cosmos/README.md

```diff
@@ -18,7 +18,7 @@ For documentation on how to use this package, please see [Azure Management Libra
 <dependency>
     <groupId>com.azure.resourcemanager</groupId>
     <artifactId>azure-resourcemanager-cosmos</artifactId>
-    <version>2.26.0</version>
+    <version>2.27.0-beta.1</version>
```

Comment:
Revert this (if still exist after refresh main). No need to update readme now. They will get updated before release.